msrest/http_logger.py

```diff
@@ -43,6 +43,8 @@ def log_request(adapter, request, *args, **kwargs):
         _LOGGER.debug("Request method: %r", request.method)
         _LOGGER.debug("Request headers:")
         for header, value in request.headers.items():
+            if header == 'Authorization':
```

Comment:
I would actually like the fact that the "Authorization" header was passed to be logged though. I'll `.lower()` it though. Not sure why I didn't add that to begin with.